Server/package.json

```diff
@@ -15,6 +15,7 @@
     "cors": "^2.8.5",
     "crypto": "^1.0.1",
     "express": "^4.17.1",
+    "mongodb": "^3.6.3",
```

Comment:
is this import necessary?
mongoose should be handle the mongoDB interactions.

Furthermore this is in the server directory. why?